src/transformers/configuration_utils.py

```diff
@@ -56,6 +56,8 @@ class PretrainedConfig(object):
                 Whether the model is used as an encoder/decoder or not.
             is_decoder (:obj:`bool`, `optional`, defaults to :obj:`False`):
                 Whether the model is used as decoder or not (in which case it's used as an encoder).
+            add_cross_attention (:obj:`bool`, `optional`, defaults to :obj:`False`):
+                Whether cross-attention layers should be added to the model. Note, this option is only relevant for models that can be used as decoder models withith the `EncoderDecoderModel` class, which consists of all models in `AUTO_MODELS_FOR_CAUSAL_LM`.
```

Comment:
Please use more lines as it gets unreadable otherwise (unless you scroll) in some editors like VS code. Also `EncoderDecoderModel` -> :class:`~transformers.EncoderDecoderModel` and `AUTO_MODELS_FOR_CAUSAL_LM` -> ``AUTO_MODELS_FOR_CAUSAL_LM``.